lib/workers/branch/parent.js

```diff
@@ -62,6 +62,13 @@ async function getParentBranch(config) {
   // Now check if PR is unmergeable. If so then we also rebase
   if (pr && pr.isConflicted) {
     logger.debug('PR is conflicted');
+
+    if (pr.canRebase && !config.rebaseConflictedPrs) {
```

Comment:
This logic should exist in the existing if pr.canRebase block below it

Reply:
Moved the logic to below block.